components/installation-telemetry/cmd/send.go

```diff
@@ -56,6 +56,11 @@ var sendCmd = &cobra.Command{
 			Set("totalWorkspaces", data.TotalWorkspaces).
 			Set("totalInstances", data.TotalInstances)
 
+		platform := os.Getenv("GITPOD_INSTALLATION_PLATFORM")
+		if platform != "" {
+			properties.Set("platform", platform)
```

Comment:
Do we want to leave the `platform` as blank in the Segment data, or do we want something like `unknown` in there?